gcemetadata/lib/gcemetadata/__init__.py

```diff
@@ -0,0 +1,268 @@
+# Copyright (c) 2015 SUSE LLC, Robert Schweikert <rjschwei@suse.com>
+#
+# This file is part of gcemetadata.
+#
+# gcemetadata is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# gcemetadata is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ec2metadata.  If not, see <http://www.gnu.org/licenses/>.
+
```

Comment:
Is there a reason why GCEMetadataException and GCEMetadata classes are implemented in one file named __init__.py ? Maybe place an empty  __init__.py to let the application know the path and put exceptions and metadata classes in its own files.